pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -217,3 +221,15 @@ func getProfileData(tunedTemplate string, data interface{}) (string, error) {
 func IsIRQBalancingGloballyDisabled(profile *performancev2.PerformanceProfile) bool {
 	return profile.Spec.GloballyDisableIrqLoadBalancing != nil && *profile.Spec.GloballyDisableIrqLoadBalancing
 }
+
+func IsRealTimeHintEnabled(profile *performancev2.PerformanceProfile) bool {
+	return profile.Spec.WorkloadHints.RealTime != nil && *profile.Spec.WorkloadHints.RealTime
```

Comment:
can `profile.Spec.WorkloadHints` be nil? we should probably add some unit tests, mostly to document the behavior

Reply:
additionally, the way we are handling the hints conflates "unspecified" with "false". We treat them the same. Is it what we want to do? possibly yes, let's just explicitely agree about it.

Reply:
#_1 done
#_2 yes, default realTime Hint cannot be unspecified because default is = true, and the other ones unspecified is = false

Reply:
So how do we handle `profile.Spec.WorkloadHints` possibly being nil?

Reply:
I didn't realize and I've removed it with the refactor but I have added id again:
[
`if profile.Spec.WorkloadHints != nil {`](https://github.com/openshift/cluster-node-tuning-operator/blob/59698befa86989be3295f825c2faa3e8c5b5d4bb/pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go#L171)

Reply:
thanks, this is good enough indeed.